docs/site/BelongsTo-relation.md

```diff
@@ -267,3 +267,105 @@ DO NOT declare
 `@repository.getter(CategoryRepository) protected categoryRepositoryGetter: Getter<CategoryRepository>`
 on constructor to avoid "Circular dependency" error (see
 [issue #2118](https://github.com/strongloop/loopback-next/issues/2118))
+
+## Querying related models
+
+LoopBack 4 has the concept of an `inclusion resolver` in relations, which helps
+to query data through an `include` filter. An inclusion resolver is a function
+that can fetch target models for the given list of source model instances.
+LoopBack 4 creates a different inclusion resolver for each relation type.
+
+The following is an example for using BelongsTo inclusion resolvers:
+
+Use the relation between `Customer` and `Order` we show above, an `Order`
+belongs to a `Customer`.
+
+After setting up the relation in the repository class, the inclusion resolver
+allows users to retrieve all addresses along with their related customers
+through the following code:
+
+```ts
+addressRepo.find({include: [{relation: 'customer'}]});
```

Comment:
Let's be consistent here, either use `order belongsTo customer` or `address belongsTo customer`. Personally I prefer `order belongsTo customer`.

Reply:
Yeah I think that would be more consistent with other tests.